incubator/hnc/test/conversion/conversion_test.go

```diff
@@ -210,22 +210,23 @@ spec:
 		FieldShouldContain(hierCRD, nsA, hierSingleton, ".spec", "allowCascadingDeletion:true")
 	})
 
-	It("should still have HC condition if it exists in v1alpha1", func() {
+	It("should still have HC condition if it exists in v1alpha1, but with a new schema", func() {
 		// Before conversion, create namespace B with a missing parent A (have to
 		// create A first and then delete it because otherwise the webhook will deny
 		// setting a non-existing namespace as parent).
 		createSampleV1alpha1Tree()
 		MustRun("kubectl delete ns", nsA)
-		FieldShouldContain(hierCRD, nsB, hierSingleton, ".status.conditions", "CritParentMissing")
+		FieldShouldContain(hierCRD, nsB, hierSingleton, ".status.conditions", "code:CritParentMissing")
```

Comment:
Why is this part of this change?

Reply:
Ah this is just making it more specific since the old schema uses `code`, although `CritParentMissing` naming with `Crit` prefix is already known as v1a1.

Reply:
sg!